llvm/lib/Support/APInt.cpp

```diff
@@ -3094,3 +3094,39 @@ void llvm::LoadIntFromMemory(APInt &IntVal, const uint8_t *Src,
     memcpy(Dst + sizeof(uint64_t) - LoadBytes, Src, LoadBytes);
   }
 }
+
+APInt APIntOps::avgFloorS(const APInt &C1, const APInt &C2) {
+  // Return floor((C1 + C2)/2)
+  assert(C1.getBitWidth() == C2.getBitWidth());
```

Comment:
(style) Assertions must have messages (same below): 
```
assert(C1.getBitWidth() == C2.getBitWidth() && "Unequal bitwidths");
```

Reply:
I know that's what the coding standards say, but it seems a bit silly when it encourages people to write a message that just says exactly the same as the condition, but in English.